### PR TITLE
Improve the compilation guide a little

### PR DESCRIPTION
**Pull request by stilic**
BTW the game builds fine with the latest Haxe version (see for example [this GitHub Actions workflow](https://haxe.org/download)).

### CODE REVIEW

**Comment by AngelWyvern** [comment]
(Apologies for the late response, broke my notification system.)
I can change it to say latest but I'd prefer to reword it to at least mention that if a person has issues with gits, to use 4.1.5, since I know people that have problems using latest, even if I don't. Mind if I ask why a lot of the individual components from the Visual Studio installer is being removed? Perhaps I should go in a virtual machine and test which ones are actually needed and which ones aren't.

**Comment by stilic** [comment]
> (Apologies for the late response, broke my notification system.) I can change it to say latest but I'd prefer to reword it to at least mention that if a person has issues with gits, to use 4.1.5, since I know people that have problems using latest, even if I don't. Mind if I ask why a lot of the individual components from the Visual Studio installer is being removed? Perhaps I should go in a virtual machine and test which ones are actually needed and which ones aren't.

Because a lot of them are useless. The compiler works fine without them.

**Comment by stilic** [comment]
> > (Apologies for the late response, broke my notification system.) I can change it to say latest but I'd prefer to reword it to at least mention that if a person has issues with gits, to use 4.1.5, since I know people that have problems using latest, even if I don't. Mind if I ask why a lot of the individual components from the Visual Studio installer is being removed? Perhaps I should go in a virtual machine and test which ones are actually needed and which ones aren't.
> 
> Because a lot of them are useless. The compiler works fine without them.

Here's my current setup (sorry for the french text, I can't really fix that):

image

image

**Comment by AngelWyvern** [comment]
Did some research, and you'd be correct. I've made improvements to the Readme guide. This pr will now be closed. Thanks for the info.
